src/routing/routes/common.tsx

```diff
@@ -1,5 +1,8 @@
 import { COMMON_ROUTES_NAMES } from '../routes.names'
 import { Home, Auth, Orders } from '../../pages'
+import { action as userActionAuth } from '../../utils/actionsAuth'
+
```

Comment:
why do you have 2 blank strings here? check your local prettier. it should use the config from the project.